docs/gallery/general/extensions.py

```diff
@@ -310,7 +310,7 @@ def __init__(self, **kwargs):
 class PotatoSack(MultiContainerInterface):
 
     __clsconf__ = {
-        'attr': 'potatos',
+        'attr': 'potatoes',
```

Comment:
actually this was intentional. I know it's misspelled but the api expects adding an 's'.